cmd/controller-manager/main.go

```diff
@@ -216,6 +221,7 @@ func main() {
 		go wait.Forever(func() { bsController.Run(workers, ctx.Done()) }, waitDuration)
 		go wait.Forever(func() { tidbInitController.Run(workers, ctx.Done()) }, waitDuration)
 		go wait.Forever(func() { tidbMonitorController.Run(workers, ctx.Done()) }, waitDuration)
+		go wait.Forever(func() { periodicityController.Run() }, periodicityDuration)
```

Comment:
```suggestion
		go wait.Forever(func() { periodicityController.Run() }, waitDuration)
```

how to update statefulsets is the implementation details of controller